File: Morning Snacks/JavaScript./StrongNumber.js
let number = 145
 let digits=number
let count = 1
let sum =0
let productFactorial = 1
while (digits!=0){
      
      let SeperatingNumbers=digits%10
          for( let index=1;index<=SeperatingNumbers;index++){
                    productFactorial*=index
                   
               }
                     sum+=productFactorial
               digits= Math.floor(digits/10)
               productFactorial =1
              
count++
}  
                     if (number == sum){
                    console.log("The number is a strong number")}
               else{console.log("The number is not a strong number")}

File: Morning Snacks/Java/StrongNumber.java
import java.util.Scanner;     
     public class StrongNumber{
     
     public static void main(String[]args){
                    
           Scanner input = new Scanner(System.in);
          System.out.print("Enter a number: ");

          int number = input.nextInt();
          
        int digits = number;
        int sum = 0;

        while (digits != 0) {

            int productFactorial = 1;
            int separatingNumber = digits % 10;

            for (int index = 1; index <= separatingNumber; index++) {
                productFactorial *= index;
            }

            sum += productFactorial;

            digits = digits / 10; 
        }
               
        if (number == sum) {
            System.out.println("The number is a strong number");
        } else {
            System.out.println("The number is not a strong number");
        }







}
}

